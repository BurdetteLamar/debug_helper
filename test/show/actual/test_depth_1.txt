---
Hash (message='test_depth_1'):
  size: 1
  default: 
  default_proc: 
  Pair 0:
    Key: Symbol :a
    Value: Hash {:b=>{:c=>"not ok"}}
