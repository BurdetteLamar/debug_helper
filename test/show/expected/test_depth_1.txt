---
Hash (size=1 message='test_depth_1'):
  size: 1
  Pair 0:
    Key: Symbol :a
    Value: Hash {:b=>{:c=>"not ok"}}
